composer.json

```diff
@@ -32,8 +32,8 @@
         "natxet/CssMin": "~3",
         "pear/archive_tar": "~1",
         "codeception/base": "^2.1.5",
-        "codeception/verify": "0.2.*",
-        "codeception/aspect-mock": "0.5.4",
+        "codeception/verify": "dev-master",
+        "codeception/aspect-mock": "~1",
```

Comment:
n.b. codeception/aspect-mock: ~2 caused failues, but ~1 worked out-of-the-box and provided us with a license in composer.json, so I did not take the time to upgrade to version 2.